apps/docs/src/app/core/component-docs/feed-list-item/examples/fli-action/fli-action-example.component.html

```diff
@@ -0,0 +1,64 @@
+<div class="fd-feed-list" aria-label="Feed List Item with user link example">
+    <fd-feed-list-item
+        authorTitle="John Doe"
+        authorLink="http://example.com/"
+        text="<h4>Long text, you can toggle</h4> view with more/less button. Lorem Ipsum is simply dummy text of the printing and typesetting industry."
+        maxRows="2"
+        ariaLabel="Feed list item example">
+        <ng-template #avatar>
+            <div class="fd-avatar fd-avatar--s fd-avatar--thumbnail fd-feed-list--link">
```

Comment:
Should use the fd-avatar NGX component here and will need an `fd-feed-list--link` directive.  We don't want the application developer to need to use the class property to apply fundamental styles to elements